rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -228,6 +256,16 @@ def _detect_relevant_trades(
                 )
                 decoded_events.append(spend_event)
 
+            if swap_data.fee_amount == ZERO:
+                try:
+                    raw_fee_amount, swap_data.order_type = self.cowswap_api.get_order_data(swap_data.order_uid)  # noqa: E501
+                    swap_data.fee_amount = asset_normalized_value(
+                        amount=raw_fee_amount,
+                        asset=swap_data.from_asset.resolve_to_crypto_asset(),
+                    )
```

Comment:
Move this out of the try. This is not supposed to raise anything right?
```suggestion
                    swap_data.fee_amount = asset_normalized_value(
                        amount=raw_fee_amount,
                        asset=swap_data.from_asset.resolve_to_crypto_asset(),
                    )
```